data-migrations/2025-03-05-0809-add-role-to-collaborator.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE projectCollaborators
```

Comment:
I realized that I had not included a place to store the project collaborator's access level!